apps/vr-tests-react-components/src/stories/Progress.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Progress } from '@fluentui/react-progress';
+import { TestWrapperDecoratorNoAnimation } from '../utilities/TestWrapperDecorator';
+
+storiesOf('Progress converged', module)
+  .addDecorator(TestWrapperDecoratorNoAnimation)
+  .addStory('Indeterminate', () => <Progress className="test-class" />, {
```

Comment:
Is the `className` prop needed in these stories?

Reply:
Oh no, forgot to delete that, will update!